packages/front-end/components/Features/ExperimentRefSummary.tsx

```diff
@@ -0,0 +1,328 @@
+import {
+  ExperimentReferenceRule,
+  FeatureInterface,
+  FeatureValueType,
+} from "back-end/types/feature";
+import ValueDisplay from "./ValueDisplay";
+import Link from "next/link";
+import {
+  ExperimentInterfaceStringDates,
+  ExperimentPhaseStringDates,
+} from "back-end/types/experiment";
+import React, { ReactElement, useState } from "react";
+import { getVariationColor } from "../../services/features";
+import ExperimentSplitVisual from "./ExperimentSplitVisual";
+import { date, datetime } from "../../services/dates";
+import { phaseSummary } from "../../services/utils";
+import { FaAngleDown, FaAngleRight } from "react-icons/fa";
+import ResultsIndicator from "../Experiment/ResultsIndicator";
+
+const percentFormatter = new Intl.NumberFormat(undefined, {
+  style: "percent",
+  maximumFractionDigits: 2,
+});
```

Comment:
If we don't specify a locale, it could be confusing to users who are in different regions where the number formatting is different than North American English because the formatting of this specific number could be different than the rest of the app.

For example, in English we may have 10.5% and in French locales it could be 10,5% but then this formatting would only be for this number. We can potentially either be explicit about the locale (e.g. `"en-US"`) or use `.toFixed(2)` instead in an effort to keep the number formatting consistent. 

Here's an example scenario:

```js
var percentFormatter = new Intl.NumberFormat('fr-FR', {
  style: "percent",
  maximumFractionDigits: 2,
});
undefined
percentFormatter.format(1.5555)
'155,55 %'
var percentFormatter = new Intl.NumberFormat('en-US', {
  style: "percent",
  maximumFractionDigits: 2,
});
undefined
percentFormatter.format(1.5555)
'155.55%'
```